### PR TITLE
Added game hub

### PR DESCRIPTION
**Pull request by hojat72elect**
Hello, 

I have added my own open source Android app to this list.

Its name is GameHub and it's a client for gamespot and IGDB APIs. Will help users to follow latest news about video games.

Please accept the PR. 

Thanks

### CODE REVIEW

**Comment by offa** [comment]
Thanks for you contribution. As far as I can tell, the application includes proprietary google services, which prevent inclusion.

**Comment by hojat72elect** [comment]
@offa 
If you mean firebase SDK and google crashlytics, I have removed both of them in latest version of the app.

The app is open source and doesn't use any proprietary APIs.

**Comment by hojat72elect** [comment]
I removed the classpath referring to google crashlytics.

**Comment by offa** [comment]
Nice, thank you!

File: README.md [hunk 1]
```diff
@@ -199,6 +199,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Husky**](https://codeberg.org/husky/husky) <sup>**[[F-Droid](https://www.f-droid.org/app/su.xash.husky)]**</sup>
 * [**monocles social**](https://codeberg.org/Arne/monocles_social) <sup>**[[F-Droid](https://www.f-droid.org/app/de.monocles.social)]**</sup>
 * [**Fedilab**](https://codeberg.org/tom79/Fedilab) <sup>**[[F-Droid](https://f-droid.org/app/fr.gouv.etalab.mastodon)]**</sup>
+* [**GameHub**](https://github.com/hojat72elect/GameHub) <sup>**[[Google Play Store](https://play.google.com/store/apps/details?id=ca.on.hojat.gamenews)]**</sup>
```
Comment (offa) [inline README.md:202]: Google Play isn't suitable here, but you should consider submitting it to [F-Droid](https://f-droid.org) too – let me know if you need some help.

```suggestion
* [**GameHub**](https://github.com/hojat72elect/GameHub)</sup>
```

**Review by offa — APPROVED** [review]

**Comment by shuvashish76** [comment]
> I have added my own **open source** Android app to this list.

@hojat72elect What is the license of your project?

**Comment by offa** [comment]
Thanks for spotting this @shuvashish76

**Comment by hojat72elect** [comment]
@offa @shuvashish76 

Thanks a lot for your points. Honestly, I don't have enough time for publishing and maintaining the app release on FDROID.
If someday in future I did that, will send another PR.

Thanks for your time.

**Pull request closed without merging**